example/inventory/group_vars/production/secrets.yml

```diff
@@ -0,0 +1,2 @@
+---
+cloudscale__api_token: ...
```

Comment:
Since the README mentions that this is meant to be encrypted, I think there should be an example or a link to the documentation about this here.

I also think some options might be good (commented out):

1. The straight forward version you have now.
2. The encrypted version (say with a link to the Ansible docs).
3. A `{{ lookup('env','CLOUDSCALE_API_TOKEN') }}` version.